images/manylinux_2_28_x86_64/Dockerfile

```diff
@@ -91,31 +92,34 @@ RUN git clone --depth 1 --branch v1.8 https://github.com/pocl/pocl.git \
     -DENABLE_POCLCC=OFF \
     -DENABLE_TESTS=OFF \
     -DENABLE_EXAMPLES=OFF \
+    -DLLC_HOST_CPU=generic \
```

Comment:
Fix the following error:
```
#9 10.85 -- Find out LLC target triple (for host x86_64-unknown-linux-gnu)
#9 10.86 -- Find out LLC host CPU with /usr/local/bin/llc
#9 10.87 CMake Error at cmake/LLVM.cmake:721 (message):
#9 10.87   LLVM could not recognize your CPU model automatically.  Please run CMake
#9 10.87   with -DLLC_HOST_CPU=<cpu> (you can find valid names with: llc -mcpu=help)
#9 10.87 Call Stack (most recent call first):
#9 10.87   CMakeLists.txt:472 (include)
#9 10.87 
#9 10.87 
#9 10.87 -- Configuring incomplete, errors occurred!
#9 10.87 See also "/pocl/build/CMakeFiles/CMakeOutput.log".
```

Reply:
`generic`results in [`Aborted (core dumped)`](https://github.com/guolinke/lightgbm-ci-docker/pull/34#issuecomment-2227394537). `"x86-64"` seems to be OK.

Reply:
Makes sense to me based on the description at https://portablecl.org/docs/html/install.html

> _`-DLLC_HOST_CPU=<something>` Defaults to auto-detection via `llc`. Run `llc -mcpu=help` for valid values. The CPU type is required to compile the “target” (kernel library) part of CPU backend._

> _This variable overrides LLVM’s autodetected host CPU at configure time. Useful when llc fails to detect the CPU (often happens on non-x86 platforms, **or x86 with CPU newer than LLVM**)._

For as long as we are still using pocl 1.8 (https://github.com/microsoft/LightGBM/issues/5596), LLVM is pinned to v11:

https://github.com/guolinke/lightgbm-ci-docker/blob/a0df140729a4006b4ff87b880e7aa53934ecc0e7/images/manylinux_2_28_x86_64/Dockerfile#L17-L23

LLVM 11.1.0 came out in February 2021 ([release history](https://releases.llvm.org/)), so I guess it's possible that GitHub's hosted runners could have a newer CPU than that.

Reply:
Thank you for the investigation!